PcsConfig/SeedDataTest.cs

```diff
@@ -15,10 +15,16 @@ public class SeedDataTest
         private const string CONFIG_ADDRESS = "http://127.0.0.1:9005/v1";
         private const string TELEMETRY_ADDRESS = "http://127.0.0.1:9004/v1";
         private const string SIMULATION_ADDRESS = "http://127.0.0.1:9003/v1";
+        private const string STORAGE_ADAPTER_ADDRESS_FORMAT = "http://{0}:9022/v1";
+        private readonly string STORAGE_ADAPTER_ADDRESS;
+        private const int TIMEOUT = 10000;
 
         public SeedDataTest()
         {
+            string host = "localhost";
```

Comment:
why need localhost here and it works with 127.0.0.1 for other services?

Reply:
fixed